apps/dw-frontend/pages/events_view.py

```diff
@@ -116,10 +115,11 @@ def show():
         st.info("No daily page views data available yet. Generate some pageview events using the Extract button to see this materialized view in action!")
 
     # Show workflow runs
+    title_with_info_icon("Events Workflows", "View the status and history of events processing workflows", "events_workflows_info")
```

Comment:
Looks like there's a title in `render_workflows_table`. We can remove that and keep this here? similarly for log & blob views.

Reply:
nice catch. Fixed.